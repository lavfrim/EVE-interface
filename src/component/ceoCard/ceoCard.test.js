import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import CEOCard from './index';

const ceoInfo = {
    name: 'name',
    birthday: "2003-03-27T13:27:00Z",
    race_id: 2,
};


describe('CEOCard should render correctly', () => {
    it('with moch ceoInfo', () => {
        const wrapper = mount(
            <CEOCard ceoInfo={ceoInfo} />
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});

