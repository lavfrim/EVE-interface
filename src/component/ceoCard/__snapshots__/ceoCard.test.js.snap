// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CEOCard should render correctly with moch ceoInfo 1`] = `
<CEOCard
  ceoInfo={
    Object {
      "birthday": "2003-03-27T13:27:00Z",
      "name": "name",
      "race_id": 2,
    }
  }
>
  <p>
    CEO name: name
  </p>
  <p>
    Birthday: 3/27/2003, 3:27:00 PM
  </p>
  <div
    className="ceo-card__race"
  >
    <p
      className="ceo-card__race-description"
    >
      Race:
    </p>
    <Loading>
      <span>
        Loading...
      </span>
    </Loading>
  </div>
</CEOCard>
`;

exports[`Main should render correctly with moch store 1`] = `
<CEOCard
  ceoInfo={
    Object {
      "birthday": "2003-03-27T13:27:00Z",
      "name": "name",
      "race_id": 2,
    }
  }
>
  <p>
    CEO name: name
  </p>
  <p>
    Birthday: 3/27/2003, 3:27:00 PM
  </p>
  <div
    className="ceo-card__race"
  >
    <p
      className="ceo-card__race-description"
    >
      Race:
    </p>
    <Loading>
      <span>
        Loading...
      </span>
    </Loading>
  </div>
</CEOCard>
`;
